go/parquet/pqarrow/file_writer.go

```diff
@@ -181,7 +188,10 @@ func (fw *FileWriter) WriteBuffered(rec arrow.Record) error {
 }
 
 // Write an arrow Record Batch to the file, respecting the MaxRowGroupLength in the writer
-// properties to determine whether or not a new row group is created while writing.
+// properties to determine whether a new row group is created or not while writing.
```

Comment:
Let's update this to say `to determine whether the record is broken up into more than one row group` and add a statement that it will always create at least one row group for the record, or something like that.

Reply:
Done in 9c4ca94137a143986e69feb07a0ab94baee7e334